frontend/src/metabase/visualizations/visualizations/WaterfallChart.jsx

```diff
@@ -5,20 +5,38 @@ import { assocIn } from "icepick";
 
 import {
   GRAPH_DATA_SETTINGS,
-  GRAPH_COLORS_SETTINGS,
   GRAPH_AXIS_SETTINGS,
   GRAPH_DISPLAY_VALUES_SETTINGS,
 } from "../lib/settings/graph";
 
+import { color } from "metabase/lib/colors";
+
 export default class WaterfallChart extends LineAreaBarChart {
   static uiName = t`Waterfall`;
   static identifier = "waterfall";
   static iconName = "waterfall";
   static noun = t`waterfall chart`;
 
   static settings = {
-    ...GRAPH_COLORS_SETTINGS,
     ...GRAPH_AXIS_SETTINGS,
+    "waterfall.increase_color": {
+      section: t`Display`,
+      title: t`Increase color`,
```

Comment:
You could make it slightly more compact by moving the title inside the widget:

```suggestion
      props: {title: t`Increase color`},
```

Reply:
Precisely what I need! Thank you.